src/app/pages/confirm-test-details/confirm-test-details.page.ts

```diff
@@ -105,6 +105,7 @@ export class ConfirmTestDetailsPage
   merged$: Observable<boolean | string>;
   slotId: string;
   idPrefix = 'confirm-test-details';
+  lessonArray: string[] = [];
```

Comment:
I don't see why we need to do this, can't we do

`*ngFor="let lesson of (pageState.lessonTheme$ | async)?.split(', ');"`

Otherwise we need to maintain two lists

Reply:
Oh, I thought lesson themes were a defined list. Nonetheless, I'm not a fan of maintaining and holding a second list.

Does this work, if we remove the `join(', ')` from the chain.  The type then becomes `lessonTheme$: Observable<string[]>;`

And where we need a string in the HTML, we can do `(pageState.lessonTheme$ | async)?.join(', ')`